code/game/objects/structures/window_frames.dm

```diff
@@ -170,6 +207,13 @@
 
 	return ITEM_INTERACT_SUCCESS
 
+/obj/structure/window_frame/handle_deconstruct(disassembled)
+	if(!disassembled)
+		grille?.use(round(sheet_amount * 0.5))
```

Comment:
This runtimes in testing.
`Runtime in code/game/objects/structures/window_frames.dm, line 212: Cannot execute 0.use().`

Same goes for 213 when deconstructing a frame:
`code/game/objects/structures/window_frames.dm, line 213: Cannot execute 0.forceMove().`